code/modules/paperwork/photography.dm

```diff
@@ -325,7 +325,7 @@ var/list/SpookyGhosts = list("ghost","shade","shade2","ghost-narsie","horror","s
 	to_chat(user, "<span class='notice'>[pictures_left] photos left.</span>")
 	icon_state = icon_off
 	on = 0
-	if(user.mind && !(user.mind.assigned_role == "Chaplain"))
+	if(user.mind && user.mind.assigned_role == "Chaplain")
 		if(prob(24))
 			handle_haunt(user)
 	spawn(64)

```

Comment:
don't expect you to fix this, but....

*ew*

Reply:
This'd be where that new `ADD_TIMER` business would come in handy, huh?

Reply:
yeahp.